app/src/main/java/com/ncnf/database/builder/EventBuilder.java

```diff
@@ -36,29 +37,29 @@
 public class EventBuilder extends DatabaseObjectBuilder<Event> {
 
     @Override
-    public Event toObject(String uuid, Map<String, Object> data) {
+    public Event toObject(String uuid, Map<String, Object> data){
         Objects.requireNonNull(data);
 
-        String ownerId = data.get(OWNER_KEY).toString();
-        String uuidStr = data.get(UUID_KEY).toString();
-        String name = (String) data.get(NAME_KEY);
-        Date date = ((Timestamp) data.get(DATE_KEY)).toDate();
-        GeoPoint location = (GeoPoint) data.get(LOCATION_KEY);
-        String address = data.get(ADDRESS_KEY).toString();
-        String typeStr = data.get(TYPE_KEY).toString();
+        String ownerId = data.getOrDefault(OWNER_KEY, "").toString();
+        String uuidStr = data.getOrDefault(UUID_KEY, "").toString();
+        String name = data.getOrDefault(NAME_KEY, "").toString();
+        Date date = ((Timestamp) data.getOrDefault(DATE_KEY, new Date())).toDate();
+        GeoPoint location = (GeoPoint) data.getOrDefault(LOCATION_KEY, new GeoPoint(0, 0));
+        String address = data.getOrDefault(ADDRESS_KEY, "").toString();
```

Comment:
I think this is a very good change, to use getOrDefault instead of get. It's cleaner

Reply:
Good but this might lead to incomplete fields so it has to be checked at some point

Reply:
This will be changed once we agree on the structure of each data model. This was just a way to avoid conflits during the development process.